webapp/client/src/pages/PauschbetragPersonBPage.js

```diff
@@ -0,0 +1,95 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+import styled from "styled-components";
+import FormHeader from "../components/FormHeader";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import { selectionFieldPropType } from "../lib/propTypes";
+import FormFieldRadio, { boldifyChoices } from "../components/FormFieldRadio";
+import Details from "../components/Details";
+
+const DetailsDiv = styled.div`
+  margin-bottom: var(--spacing-04);
+`;
+
+export default function PauschbetragPersonBPage({
+  stepHeader,
+  form,
+  fields,
+  prevUrl,
+}) {
+  const { t } = useTranslation();
+  const boldChoices = boldifyChoices(fields.personBWantsPauschbetrag.options);
```

Comment:
I like! Can you try to move this to an extra lib instead of having it at the react component? :grinning: